resources/assets/js/components/ckeditor.js

```diff
@@ -0,0 +1,20 @@
+/*jslint browser: true*/
+
+window.CKEDITOR_BASEPATH = '/plugins/ckeditor/';
+
+var selector = $('.js-ckeditor'),
+    ckeditor_script = CKEDITOR_BASEPATH + 'ckeditor.js',
```

Comment:
Identifier 'ckeditor_script' is not in camel case.<br>'CKEDITOR_BASEPATH' is not defined.